AuthenticationService/src/main/java/org/energygrid/east/authenticationservice/controller/ApplicationContextUtils.java

```diff
@@ -0,0 +1,21 @@
+package org.energygrid.east.authenticationservice.controller;
+
+import org.springframework.beans.BeansException;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ApplicationContextUtils implements ApplicationContextAware {
+
+    private static ApplicationContext ctx;
+
+    @Override
+    public void setApplicationContext(org.springframework.context.ApplicationContext applicationContext) throws BeansException {
+        ctx = applicationContext;
+    }
+
+    public static ApplicationContext getCtx() {
+        return ctx;
+    }
+}
```

Comment:
Waar is deze file eigenlijk voor?

Reply:
Deze wordt gebruikt door de consumers. Dit bestand is verplaatst naar de "rabbit" map omdat het niet logisch is om deze in de controller te zetten.